tests/ConfigSchema.js

```diff
@@ -0,0 +1,72 @@
+/*!
+ * Copyright (c) 2018-2020 Digital Bazaar, Inc. All rights reserved.
+ */
+
+'use strict';
+
+export const edvConfig = {
+  title: 'EDV Configuration',
+  type: 'object',
+  // TODO: do not require primary `keyAgreementKey` and `hmac` in the future
```

Comment:
OH NO! A new TODO?  :) 
What is this about?  Is there an issue for it, can we resolve it now?

Reply:
This was in the original one that I copied from. I wasn't sure if this is a change we want to make now...

Reply:
`keyAgreementKey` and `hmac` should be optional in the config.